hunting-buddy.lic

```diff
@@ -26,6 +26,8 @@ class HuntingBuddy
     @stop_on_high_threshold = @settings.stop_on_high_threshold
     echo("  @stop_on_high_threshold: #{@stop_on_high_threshold}") if $debug_mode_hunting
 
+    @after_hunt_room = @settings.force_after_hunt_room || @settings.safe_room
```

Comment:
💡 Please add an echo after line 29 like like 27 to show what the variable value ended up being. Helpful for troubleshooting. Thanks